src/components/Spinner/spinner.scss

```diff
@@ -0,0 +1,8 @@
+.imageWrap {
```

Comment:
CSS: lowercased with hyphens please

Reply:
we discussed, we will keep pascalCase in this PR